assets/styles/app.css

```diff
@@ -0,0 +1,324 @@
+body {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+.page {
+  display: flex;
+  flex-direction: column;
+  margin: unset;
+  overflow: auto;
+}
+
+.header {
+  display: flex;
+  justify-content: flex-end;
+  padding: 5%;
+}
+
+.middle {
+  color: #36b37e;
+}
+
+.description {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+}
+
+.button {
+  color: #172b4d;
+  margin-top: 100px;
+  justify-content: center;
+}
+
+.lines {
+  justify-content: center;
+  font-size: 40px;
+  line-height: 52px;
+  margin-top: unset;
+  margin-bottom: unset;
+  color: #172b4d;
+}
+
+.socials {
+  display: flex;
+  gap: 1rem;
+  justify-content: center;
+  list-style-type: none;
+}
+
+.menu {
+  display: flex;
+  justify-content: right;
+  padding: 40px;
+}
+
+.headline {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding: 20px;
+  text-align: center;
+}
+
+.background-image {
+  position: absolute;
+  width: 100%;
+}
+
+.hdescription {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  text-align: center;
+  color: #172b4d;
+}
+
+.hdescription::after {
+  content: '';
+  width: 48px;
+  height: 4px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 24px;
+  background: #36b37e;
+  border-radius: 24px;
+}
+
+.categories {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  text-align: center;
+  color: #3a4a42;
+  font-size: larger;
+}
+
+.image-container {
+  display: flex;
+  justify-content: center;
+  max-height: 500px;
+}
+
+.article {
+  display: flex;
+  justify-content: center;
+  gap: 2rem;
+  text-align: start;
+}
+
+.read {
+  position: relative;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  flex-direction: column;
+  padding: 20px;
+  text-align: center;
+}
+
+.item {
+  display: flex;
+  padding: 5px;
+  font-size: x-small;
+  color: #3a4a42;
+  background: #ebf0ee;
+}
+
+.spoon {
+  position: absolute;
+  background: white;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  border-radius: 12px;
+  margin: 0 25px 1px 25px;
+  padding-left: 10px;
+  padding-right: 10px;
+}
+
+.button-tag {
+  padding: 5px;
+  background: #36b37f;
+  color: white;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  border: none;
+}
+
+.article-container {
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+}
+
+.text-aling {
+  display: flex;
+  justify-content: center;
+  padding-top: 50px;
+}
+
+.paragraf {
+  display: flex;
+  justify-content: center;
+}
+
+.plane {
+  padding: 20px;
+  margin-left: -20px;
+  margin-bottom: -60px;
+}
```

Comment:
- [ ] I will suggest that you take time to rename your class names to reflect what the element is all about.